Orange-OpenSource/Orange-Boosted-Bootstrap
# 🚀 Release 5.1.1

- [x] Merge remaining commits from Bootstrap
  - https://github.com/twbs/bootstrap/commit/aa06dffdf65f15f6663159c9785432b82adde20f
  - https://github.com/twbs/bootstrap/commit/23b3d636e10de3b00cd717e7fd32123bde95d0fe
  - https://github.com/twbs/bootstrap/commit/b6855ae13817f516f63bd2916ead5243e07f210b
  - ~~https://github.com/twbs/bootstrap/commit/0d81d3cbc14dfcdca8a868e3f25189a4f1ab273c~~  

---

  - Firefox ESR
  - IE11 (v4 only)
  - Latest Edge, Chrome, Firefox, Safari
  - iOS Safari
  - Chrome & Firefox on Android
  - check wrong matches in `CHANGELOG.md`, and maybe `site/content/docs/<version>/migration.md`
  - :warning: check the `package-lock.json` content, only "boosted" should have its version changed!
- ~~if year changed recently, happy new year :tada: but please change © year in `.scss` main files (reboot, grid, utilities and main file) as well as in `NOTICE.txt`.~~
  - install [Conventionnal Changelog](https://github.com/conventional-changelog/conventional-changelog) and `conventional-changelog-cli` globally
  - run `conventional-changelog -p angular -i CHANGELOG.md -s`
  - and probably maintain [a ship list (eg for v4.4.0)](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/226)
- [ ] merge on `main`
  - attach zip file
  - paste CHANGELOG / Ship list in the release's description
- ~~run `build/ship.sh x.x.x` to… (v4 only)~~
  - if you are already logged in NPM (with a personnal account, for example), [you'd better use a repository scoped `.npmrc` file](https://stackoverflow.com/questions/30114166/how-to-have-multiple-npm-users-set-up-locally)
  - if you're releasing a pre-release, use `--tag`, eg for v5-alpha1 `npm publish boosted-5.0.0-alpha1.tgz --tag next`
  - [ ] copy `../_site` (v5-dev) or `../bs-docs` (v4-dev) to the `gh-pages` branch
